### DISCUSSION TITLE
Upgrade Path from Beta

### DISCUSSION THREAD

**Discussion by jwmasekre** [Security-Onion-Solutions/securityonion#11036] [2.4] [answered]
I didn't see anything explicitly calling it out, just that it didn't exist, but is the upgrade path from 2.4.3 to 2.4.4+ just to reinstall the OS? I was hoping to do an in place upgrade since we were testing a distributed setup and there's some decent overhead with reinstallation, but if that's the path we need to go I definitely want to know now so I don't waste anymore time trying to find a solution that doesn't involve rebuilding.

**Comment by dougburks** [comment, marked as answer]
From the 2.4.3 blog post at https://blog.securityonion.net/2023/07/security-onion-24-beta-4-release-now.html:
> Upgrades from this 2.4 Beta release to anything else will not be supported.

**Reply by jwmasekre** [reply]
Okay, so to confirm then the expected path is reinstall?

**Reply by dougburks** [reply]
Correct.
